pkgs/applications/science/logic/abc/libabc.nix

```diff
@@ -30,6 +30,6 @@ stdenv.mkDerivation rec {
     homepage    = "https://people.eecs.berkeley.edu/~alanmi/abc/abc.htm";
     license     = stdenv.lib.licenses.mit;
     platforms   = stdenv.lib.platforms.unix;
-    maintainers = [ stdenv.lib.maintainers.thoughtpolice ];
+    maintainers = [ ];
```

Comment:
@kwquick that's about the extent of it, if you're still up for it please consider adding yourself as the maintainer.